saleor/payment/gateways/adyen/webhooks.py

```diff
@@ -74,8 +75,8 @@ def get_payment(
         logger.warning("Missing payment ID. Reference %s", transaction_id)
         return None
     try:
-        _type, db_payment_id = from_global_id(payment_id)
-    except (UnicodeDecodeError, binascii.Error):
+        _type, db_payment_id = from_global_id_or_error(payment_id)
```

Comment:
we are inconsistent now as we used `type_` elsewhere and `_type` here.

Reply:
Isn't it fine though? Underscore in front suggests that this variable is not used anywhere later in the code. It could also be changed into `_` since it is not used anywhere.